[2022-01-30 17:42:29,904] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-02-02T06:00:00+00:00 [queued]>
[2022-01-30 17:42:29,910] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-02-02T06:00:00+00:00 [queued]>
[2022-01-30 17:42:29,911] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 17:42:29,911] {taskinstance.py:1239} INFO - Starting attempt 3 of 3
[2022-01-30 17:42:29,911] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 17:42:29,919] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2021-02-02 06:00:00+00:00
[2022-01-30 17:42:29,925] {standard_task_runner.py:52} INFO - Started process 8404 to run task
[2022-01-30 17:42:29,929] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-02-02T06:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpt9bfy8v3', '--error-file', '/tmp/tmp2vbkrkof']
[2022-01-30 17:42:29,929] {standard_task_runner.py:77} INFO - Job 43: Subtask ingest
[2022-01-30 17:42:29,981] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-02-02T06:00:00+00:00 [running]> on host 3a2cf9f2a065
[2022-01-30 17:42:30,024] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-02-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-02T06:00:00+00:00
[2022-01-30 17:42:30,025] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-30 17:42:30,025] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-01-30 17:42:30,032] {subprocess.py:85} INFO - Output:
[2022-01-30 17:42:30,036] {subprocess.py:89} INFO - ***-worker.pid
[2022-01-30 17:42:30,036] {subprocess.py:89} INFO - ***.cfg
[2022-01-30 17:42:30,036] {subprocess.py:89} INFO - dags
[2022-01-30 17:42:30,036] {subprocess.py:89} INFO - logs
[2022-01-30 17:42:30,036] {subprocess.py:89} INFO - output.csv
[2022-01-30 17:42:30,036] {subprocess.py:89} INFO - output_2021-02.csv
[2022-01-30 17:42:30,037] {subprocess.py:89} INFO - plugins
[2022-01-30 17:42:30,037] {subprocess.py:89} INFO - requirements.txt
[2022-01-30 17:42:30,037] {subprocess.py:89} INFO - webserver_config.py
[2022-01-30 17:42:30,037] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-30 17:42:30,060] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20210202T060000, start_date=20220130T174229, end_date=20220130T174230
[2022-01-30 17:42:30,107] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-30 17:42:30,135] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
